### PR TITLE
A category filter is now available when searching events

### PR DESCRIPTION
**Pull request by rpg600**

### CODE REVIEW

File: app/Resources/views/search/_filters.html.twig [hunk 1]
```diff
@@ -1,30 +1,44 @@
-<div class="search__bar background--blue space--30 text--white text--body">
-    <div class="search__bar__options">
-        <input class="search-box"name="{{ constant('AppBundle\\Search\\SearchParametersFilter::PARAMETER_QUERY') }}" placeholder="Recherche">
+<div class="background--blue space--30 b__nudge--bottom-60 text--white text--body">
+    <div class="search__bar">
+        <div class="search__bar__options">
+            <input class="search-box"name="{{ constant('AppBundle\\Search\\SearchParametersFilter::PARAMETER_QUERY') }}" placeholder="Recherche">
```
Comment (mykiwi) [inline app/Resources/views/search/_filters.html.twig]: missing space
```diff
-class="search-box"name=
+class="search-box" name=
```

**Comment by mykiwi** [comment]
What about store `AppBundle\\Search\\SearchParametersFilter` in a template variable? And use it everywhere?

**Comment by rpg600** [comment]
@mykiwi And you want to concat this variable with the constant name ? it will be ugly anyway.

**Comment by mykiwi** [comment]
Better idea, what about `constant('...', search)` ?

**Comment by rpg600** [comment]
@mykiwi Yep !

**Pull request merged**